dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/features/ServerApi.java

```diff
@@ -68,21 +68,19 @@
    @GET
    @Path("/server")
    @ResponseParser(ParseServers.class)
-   @Fallback(Fallbacks.EmptyIterableWithMarkerOnNotFoundOr404.class)
```

Comment:
This is a behavior change since this method will now fail upon 4xx responses, when previously it just returned an empty list. Consider adding the corresponding fallback and keep returning empty lists.

Reply:
EmptyIterableWithMarkerOnNotFoundOr404 only affects responses with 404 code or triggering ResourceNotFoundException, none of which if possible for CloudControl list APIs.
This means that previously the method was failing (as expected) for all 4xx responses, ie. there is no behavior change.